bundles/model/org.eclipse.smarthome.model.rule/src/org/eclipse/smarthome/model/rule/Rules.xtext

```diff
@@ -41,12 +44,24 @@ CommandEventTrigger:
 	'Item' item=ItemName 'received command' (command=ValidCommand)?
 ;
 
+GroupMemberCommandEventTrigger:
+    'Member' 'of' group=ItemName 'received command' (command=ValidCommand)?
```

Comment:
I like this `Member of ...´-version much better too! 

Independently of the naming one general remark though - this adds `Member` and `of`  as new keywords to the language. Therefore this it is breaking all existing rule scripts which are using these words for anything like variable names. This isn't necessarily a problem - just everybody should at least be aware of this.

Reply:
Maybe we can lower the conflict potential by defining just a single keyword, like `MemberOf`?

Reply:
That looks awkward though. To be honest, I really like the proposal as it is.

Reply:
@sjka Wouldn't it be nicer though to have `'Member of'`, just like we have `'received command'`? As after "Member" there always has to be the word "of", this would make the content completion quicker and easier to understand.

Reply:
Afair, having spaces in keywords is considered bad style (whitespace there suddenly matters and the tokenizer being greedy). However, as you pointed out, this is not the first place where it was done like that, therefore: why should be care now, right? 
This would indeed minimized the risk of collisions then.